data/ecosystems/o/osmosis.toml

```diff
@@ -1,9 +1,43 @@
 # Ecosystem Level Information
 title = "Osmosis"
 
-sub_ecosystems = ["Apollo DAO", "Eve Network", "ZKValidator"]
-
-github_organizations = ["https://github.com/osmosis-labs"]
+sub_ecosystems = [
+  "Apollo DAO",
```

Comment:
A lot of these look more like Cosmos ecosystem or cross-chain.  We are going to be exposing secondary ecosystems in the taxonomy shortly, but these aren't direct sub ecosystems of Osmosis at the moment. 

Looking through them, I would remove most of these.

For Mars, is it possible to find the repos dedicated to Osmosis and put them in Osmosis directly?

Reply:
Sure.  It gets a bit tricky in the Cosmos architecture, as all of the linked projects are building for/on Osmosis blockchain in some way, but just doing their interactions via IBC (for example Quasar is vaults system for Osmosis, but it triggers the vaults over IBC).

Reply:
But sure, I can remove the ones that are not Osmosis-focused